docs/spec_draft.md

```diff
@@ -627,6 +628,51 @@ matrix, then the behavior is undefined.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.clamp
+
+### Semantics
+
+Clamps every element of the `operand` tensor between a minimum and maximum
+value and produces a `result` tensor. More formally, `result[i0, ..., iR-1]` =
+`minimum(maximum(operand[i0, ..., iR-1], min_val), max_val)`,
+where `min_val = rank(min) == 0 ? min : min[i0, ..., iR-1]`,
+`max_val = rank(max) == 0 ? max : max[i0, ..., iR-1]`, `minimum` and `maximum`
```

Comment:
I like the `_val` convention. Let's change the spec for `select` to also use it, i.e. to say `pred_val` instead of `predicate`.

Reply:
Sure, thanks.